docs/excluding-from-management.md

```diff
@@ -0,0 +1,79 @@
+# Excluding from Management
+
+In many cases, you may find it useful to exclude resources from being managed. This could be because your tenant has a large number of a particular resource and thus operationally burdensome to manage. Other times, you may wish to exclude them as your development workflow only pertains to a specific subset of resources and you’d like to omit all other resources for performance. Regardless of the use case, there are several options available for expressing exclusions in the Deploy CLI.
+
+## Excluding entire resources by type
+
+For more complex tenants, you may find yourself wanting to omit entire resource types. Some common use cases for wanting this capability:
+
+- Enterprise tenant with thousands of organizations, managing all would be operationally burdensome
+- CI/CD process only focuses on managing roles, you may wish to exclude all others
+- Feature development pertains to hook, you may wish to temporarily exclude all others to optimize performance
+
+This type of exclusion is expressed by passing an array of resource names into the `AUTH0_EXCLUDED` configuration property. This exclusion works **bi-directionally**, that is, both when export from Auth0 and importing to Auth0, regardless if resource configuration files exist or not.
+
+All supported resource values for exclusion:
+
+```
```

Comment:
In https://github.com/auth0/auth0-deploy-cli/blob/2b0ccdb7cd8934393f2630486286371b6e58547d/docs/configuring-the-deploy-cli.md#auth0_excluded you're also displaying all possible array values, but formatted in such a way that they are all visible without having to scroll horizontally. Maybe use the same formatting here?